src/Crypto/Secp256k1.hs

```diff
@@ -395,6 +479,17 @@ exportSignatureDer (Signature fptr) = unsafePerformIO $ do
         unsafePackByteString (outBuf, len)
 
 
+-- | Convert signature to a normalized lower-S form. 'Nothing' indicates that it
+-- was already normal.
+normalizeSignature :: Signature -> Maybe Signature
```

Comment:
It seems like it would be more ergonomic to have the signature `Signature -> (Bool, Signature)` where the bool signals whether the signature return parameter is the original or a modified signature.

Reply:
Changed it. I also thought it was a bit awkward when copying it over from `secp256k1-haskell`.